src/Http/Controllers/PaymentCallbackController.php.stub

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Solutionplus\Payable\Http\Controllers;
```

Comment:
**Blocking**

this namespace should reflect the actual namespace in the project after publishing this file